### PR TITLE
fixes the issue where if clicked the page would stop displaying info…

### LINKED ISSUE #1106: Strange behavior on 'Events' page
### Overview
On the 'Events' page, clicking on some the text "Meetup Events" or "Online Project Team Meetings" causes them to break when the viewport width is greater than 767px.

### Action Items
Disable the dropdown effect when width is greater than 767px.

### Resources/Instructions
demo

### PR DESCRIPTION
**Pull request by thatguyish**
Fixes #1106

…rmation (like meetup events, and team meetings)

### CODE REVIEW

**Review by jbubar — CHANGES_REQUESTED** [review]
the cute little carrot animation on mobile seemed to have broke. Here is what it looks like on the live page.. I think you might need to add back in the this.classList.toggle("active") some how
<details>
<summary>live site</summary>

</details>

File: events/events.html [hunk 1]
```diff
@@ -66,13 +66,24 @@ <h2 class="event-title-1">
...
+      let dropDown = this.nextElementSibling;
+      if (dropDown.style.display === "block") {
+        dropDown.style.display = "none";
+      } else {
+        dropDown.style.display = "block";
+      }
+    }
+  }
+  window.addEventListener('resize',fixDropDown)
+  function fixDropDown(){
```
Comment (akibrhast) [inline events/events.html]:
```javascript
function fixDropdown()
```
Please rename your function to what it does and not what it fixes

**Review by akibrhast — CHANGES_REQUESTED** [review]

**Review by jbubar — APPROVED** [review]
Looks good!

**Review by jbubar — CHANGES_REQUESTED** [review]
Oops. just noticed something...
When you resize from desktop to mobile.. the accordions should start out closed

**Comment by thatguyish** [comment]
@jbubar okay cool thank you 😁 will make hopefully final changes lol.

**Comment by thatguyish** [comment]
@jbubar should be good lol sorry about that.

**Pull request merged**